Started on:   Sat Jun  7 10:40:40 CEST 2014
COMPILING: javac *.java
Type is: unmodified
Executing on Haswell: ssh htmsql@deskkemper05.informatik.tu-muenchen.de -i ~/develop/private-key-protected.openssh source ~/develop/mysql/util-mysql.sh ; start_mysql unmodified ;
STARTING DATABASE: /home/htmsql/develop/mysql/mysql-5.6.10-unmodified/install/bin/mysqld --no-defaults 		--max_connections=1000 		--performance_schema=OFF 		--table_open_cache_instances=32 		--query_cache_type=0 		--query_cache_size=0 		--innodb_buffer_pool_size=2684354560 		--innodb_log_file_size=268435456 		--innodb_log_buffer_size=16777216 		--innodb_flush_method=fsync 		--innodb_flush_log_at_trx_commit=2
/home/htmsql/develop/util.sh: line 59: /home/htmsql/develop/mysql/mysql-5.6.10-unmodified/install/bin/mysqld: No such file or directory
INITIALIZING: java -Dconfigure=./config.txt -cp .:./mysql-connector-java-5.1.11-bin.jar DBGen
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1119)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2257)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:784)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:354)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:284)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:187)
	at Common.getDBConnection(Common.java:57)
	at DBGen.generate(DBGen.java:18)
	at DBGen.main(DBGen.java:71)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1119)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:343)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2178)
	... 14 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at java.net.Socket.connect(Socket.java:528)
	at java.net.Socket.<init>(Socket.java:425)
	at java.net.Socket.<init>(Socket.java:241)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:253)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:292)
	... 15 more
output: /home/htmsql/develop/mysql/report-unmodified-txbench/1-20140607_1040-pr_r75_u25-t4-d150
Executing on Haswell: ssh htmsql@deskkemper05.informatik.tu-muenchen.de -i ~/develop/private-key-protected.openssh source ~/develop/mysql/util-mysql.sh ; profile_mysql unmodified /home/htmsql/develop/mysql/report-unmodified-txbench/1-20140607_1040-pr_r75_u25-t4-d150 & ;
RUNNING TXBENCH: java -Dconfigure=./config.txt -cp .:./mysql-connector-java-5.1.11-bin.jar SibenchClient 4 75 25 S2PL 150 > /home/htmsql/develop/mysql/report-unmodified-txbench/1-20140607_1040-pr_r75_u25-t4-d150/result.txt
/home/htmsql/develop/util.sh: eval: line 59: syntax error near unexpected token `;'
/home/htmsql/develop/util.sh: eval: line 59: `ssh htmsql@deskkemper05.informatik.tu-muenchen.de -i ~/develop/private-key-protected.openssh source ~/develop/mysql/util-mysql.sh ; profile_mysql unmodified /home/htmsql/develop/mysql/report-unmodified-txbench/1-20140607_1040-pr_r75_u25-t4-d150 & ;'
waiting for run_pid 26885
com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1119)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2257)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:784)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:354)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:284)
	at java.sql.DriverManager.getConnection(DriverManager.java:571)
	at java.sql.DriverManager.getConnection(DriverManager.java:187)
	at Common.getDBConnection(Common.java:57)
	at SibenchClient.init(SibenchClient.java:79)
	at SibenchClient.main(SibenchClient.java:256)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1119)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:343)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2178)
	... 14 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at java.net.Socket.connect(Socket.java:528)
	at java.net.Socket.<init>(Socket.java:425)
	at java.net.Socket.<init>(Socket.java:241)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:253)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:292)
	... 15 more
Executing on Haswell: ssh htmsql@deskkemper05.informatik.tu-muenchen.de -i ~/develop/private-key-protected.openssh source ~/develop/profiling.sh ; stop_profiling /home/htmsql/develop/mysql/report-unmodified-txbench/1-20140607_1040-pr_r75_u25-t4-d150 ;
Run was interrupted by Control-C.
run: no process found
perf: no process found
Profiling stopped
File /perf.stat does not exist
File /perf.stat.events does not exist
File /perf.stat.rtm-events does not exist
Profiling stopped
File /home/htmsql/develop/mysql/report-unmodified-txbench/1-20140607_1040-pr_r75_u25-t4-d150/perf.stat does not exist
File /home/htmsql/develop/mysql/report-unmodified-txbench/1-20140607_1040-pr_r75_u25-t4-d150/perf.stat.events does not exist
File /home/htmsql/develop/mysql/report-unmodified-txbench/1-20140607_1040-pr_r75_u25-t4-d150/perf.stat.rtm-events does not exist
results: No such file or directory
cat: /home/htmsql/develop/mysql/report-unmodified-txbench/1-20140607_1040-pr_r75_u25-t4-d150/my-perf.stat: No such file or directory
Error executing script
Profiling stopped
File /perf.stat does not exist
File /perf.stat.events does not exist
File /perf.stat.rtm-events does not exist
